src/map/clif.cpp

```diff
@@ -2810,7 +2810,7 @@ void clif_additem( map_session_data *sd, int n, int amount, unsigned char fail )
 #if PACKETVER >= 20150226
 		clif_add_random_options( p.option_data, sd->inventory.u.items_inventory[n] );
 #if PACKETVER >= 20160921
-		p.favorite = sd->inventory.u.items_inventory[n].favorite;
+		p.favorite = ( sd->inventory.u.items_inventory[n].favorite != 0 ) ? 1 : 0;
```

Comment:
```suggestion
		p.favorite = ( sd->inventory.u.items_inventory[n].favorite != 0 ) ? true : false;
```

Reply:
That can't be, favorite from `PACKET_ZC_ITEM_PICKUP_ACK` is uint8
https://github.com/rathena/rathena/blob/master/src/map/packets_struct.hpp#L572

Reply:
you're right sorry, I mistook then with PACKET_ZC_INVENTORY_TAB p.favorite